.github/workflows/virtualbox_image.yml

```diff
@@ -0,0 +1,97 @@
+name: Create VirtualBox disk image from Dockerfile
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  push:
+    paths:
+      - 'docker/*'
+      - '.github/workflows/virtualbox_image.yml'
+
+env:
+  VM_PASSWORD: temppwd
+  DOCKER_IMAGE_BASENAME: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3
+  DOCKER_IMAGE_TAG: :refs_heads_${{ github.ref_name }}-bk0
+
+jobs:
+  # pyne_image_build:
+  #   runs-on: ubuntu-latest
+  #   outputs:
+  #     image_tag: ${{ steps.output_tag.outputs.image_tag }}
+
+  #   steps:
+  #     - name: Checkout repository
+  #       uses: actions/checkout@v3
+
+  #     - name: Log in to the Container registry
+  #       uses: docker/login-action@v2
+  #       with:
+  #         registry: ghcr.io
+  #         username: ${{ github.repository_owner }}
+  #         password: ${{ secrets.GITHUB_TOKEN }}
+
+  #     - name: Build PyNE docker image
+  #       uses: firehed/multistage-docker-build-action@v1
+  #       id: build_pyne
+  #       with:
+  #         repository: ${{ env.DOCKER_IMAGE_BASENAME }}
+  #         stages: base_python, moab, dagmc, openmc
+  #         server-stage: pyne
+  #         quiet: false
+  #         tag-latest-on-default: false
+  #         dockerfile: docker/ubuntu_22.04-dev.dockerfile
+
+  #     - id: output_tag
+  #       run: |
+  #         echo "image_tag=$DOCKER_IMAGE_TAG" >> $GITHUB_OUTPUT
+
+  # BuildTest:
+  #   needs: [pyne_image_build]
+  #   runs-on: ubuntu-latest
+
+  #   container:
+  #     image: ghcr.io/${{ github.repository_owner }}/pyne_ubuntu_22.04_py3/pyne${{ needs.multistage_image_build.outputs.image_tag }}
+  #   steps:
+  #     - name: Checkout repository
+  #       uses: actions/checkout@v3
+
+  #     - name: use BuildTest composite action
+  #       uses: ./.github/actions/build-test
+  #       with:
+  #         stage: ${{ matrix.stage }}
+  #         hdf5: ${{ matrix.hdf5 }}
+      
+  virtualbox_image_build:
+    # needs: [BuildTest]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      # - name: Set up QEMU
+      #   uses: docker/setup-qemu-action@v2
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build VirtualBox image using d2vm
+        run: |
+          sudo apt-get update
+          sudo apt-get install qemu-utils
```

Comment:
You may want to go ahead and install the full list of dependencies

Reply:
After adding all the dependencies, it seems like `d2vm` has [successfully](https://github.com/bquan0/pyne/actions/runs/5772596807/job/15647740501) converted an existing image from ghcr.io.